### ISSUE TITLE
With X18Se  on 1.5 FCC  Trims do  not work 

### ISSUE THREAD

**Issue by fhpage** [FrSkyRC/ETHOS-Feedback-Community#2962]
On X18SE Trim Mix broke...  no to little movement until MAX or MIN reached, then some movement...
Apparently by adding a comment this issue was closed by me... So opened another.

**Comment by bsongis-frsky** [comment]
I have just tested on X20S Simulator and it works perfectly. Checking on the real radio!

**Comment by bsongis-frsky** [comment]
On the radio the speed is not reliable, but it seems to be working right!

**Comment by fhpage** [comment]
"Speed is not reliable";   Yes, clicks are not linear.   "but it seems to be working right";   Not my idea of "Right".
I did another test. 
RESULTS:
This is with 1.5 nightly.
Using "Airplane" mixes.
Set the aileron trim Range to 25% and the Step to Coarse. 32 clicks put the trim at 25%. This is what one would expect. 
Next, created a Trim mix. *Set the Range to 25% and the Step to Coarse.* 
Results:
Clicks   Output %
0     0.2
22   1.8
23    2.5
25   3.3
36    4.1
Reversed the direction
-24   3.3
-26   2.5
-30   1.8
-36    1
-39    .2
-48    -0.6

I checked the output numbers with a signal meter. They are accurate.
Again, this is with a range of 25% and the Step is Coarse.
*The Trim mix is NOT usable.**
*
